aloscene/mask.py

```diff
@@ -13,20 +22,113 @@ class Mask(aloscene.tensors.SpatialAugmentedTensor):
     """
 
     @staticmethod
-    def __new__(cls, x, *args, **kwargs):
+    def __new__(cls, x, labels: Union[dict, Labels] = None, *args, **kwargs):
         # Load frame from path
         if isinstance(x, str):
             x = load_mask(x)
-            kwargs["names"] = ("C", "H", "W")
+            kwargs["names"] = ("N", "H", "W")
         tensor = super().__new__(cls, x, *args, **kwargs)
+        tensor.add_label("labels", labels, align_dim=["N"], mergeable=True)
+        tensor.add_property("mask_size", x.shape)
+        tensor.add_property("is_mask", not isinstance(labels, Labels))
```

Comment:
Does `is_mask` make sense for a class with the name `Mask` ? 

The following is weird
```
mask = Mask()
mask.is_mask # False
```

Reply:
OK. I removed this property also